src/hooks/charts/useChartThrottledPoints.js

```diff
@@ -101,10 +104,17 @@ export default function useChartThrottledPoints({
   // Only show the chart if we have chart data, or if chart data is still loading
   const showChart = useMemo(
     () =>
-      throttledPoints?.points.length > 5 ||
-      throttledPoints?.points.length > 5 ||
-      (fetchingCharts && !isFetchingInitially),
-    [fetchingCharts, isFetchingInitially, throttledPoints]
+      (nativeCurrency !== 'ETH' || asset?.address !== 'eth') &&
```

Comment:
for the address you should use `import {  ETH_ADDRESS } from '@rainbow-me/references';`